sdk/python/feast/inference.py

```diff
@@ -45,7 +45,7 @@ def update_entities_with_inferred_types_from_feature_views(
 
                 # get entity information from information extracted from the view batch source
                 extracted_entity_name_type_pairs = list(
-                    filter(lambda tup: tup[0] == entity.join_key, col_names_and_types,)
+                    filter(lambda tup: tup[0] == entity.join_key,col_names_and_types,)
```

Comment:
any reason why there are changes here? and the next section?

Reply:
ahh my bad, it's my formatter